umi-tools.wdl

```diff
@@ -111,6 +113,7 @@ task Dedup {
         inputBamIndex: {description: "The index for the ipnut BAM file.", cateogry: "required"}
         outputBamPath: {description: "The location to write the output BAM file to.", category: "required"}
         statsPrefix: {description: "The prefix for the stats files.", category: "advanced"}
+        umiSeparator: {description: "Seperator for UMI sequence, default with '_'", category: "advanced"}
```

Comment:
There is no need to mention defaults in the descriptions, WDL-AID will get those automatically.